src/components/application_manager/include/application_manager/app_service_manager.h

```diff
@@ -65,7 +62,7 @@ class AppServiceManager {
    * @param app_manager
    */
   AppServiceManager(ApplicationManager& app_manager,
-                    resumption::LastState& last_state);
+                    resumption::LastStateWrapperPtr last_state);
```

Comment:
@YarikMamykin looks like its a public function. Should we deprecate this?

Reply:
@AKalinich-Luxoft deprecated in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/e4b159464b7b99d6cffd74f6c4a263b2a7243026